sklearn/preprocessing/_label.py

```diff
@@ -129,7 +129,7 @@ def transform(self, y):
             Labels as normalized encodings.
         """
         check_is_fitted(self)
-        y = column_or_1d(y, dtype=self.classes_.dtype, warn=True)
+        y = column_or_1d(y, dtype=self.classes_.dtype, warn=True)  # Pytorch breaks
```

Comment:
@ogrisel @betatim I think we might need to handle LabelEncoder before we can continue with f1_score.